_data/security.yml

```
@@ -61,6 +61,13 @@ websites:
       - hardware
     exception: "Requires X.509-capable cryptotoken or smartcard."
 
+  - name: Check Point
```

```suggestion
  - name: Check Point CloudGuard Dome9
```

2FA only seems to apply to this specific product (I believe it has a separate auth system as well https://secure.dome9.com/v2/login rather than the main Check Point account at https://accounts.checkpoint.com/)